release/release_workflow.wdl

```diff
@@ -0,0 +1,346 @@
+     git checkout develop
+     ${updateVersionCommand}
+     git add .
+     git diff-index --quiet HEAD || git commit -m "Update ${repo} version from ${releaseV} to ${nextV}"
+     git push ${true="--dry-run" false="" dryRun} origin develop
+     
+     pwd > executionDir.txt
+   }
+
+   output {
+     String version = releaseV
+     String executionDir = read_string(repo + "/executionDir.txt")
+   }
+}
+
+task wait_for_artifactory {
+    String repo
+    String version
+    
+    command <<<
+        checkIfPresent() {
+            isPresent=$(curl -s --head https://artifactory.broadinstitute.org/artifactory/simple/libs-release-local/org/broadinstitute/${repo}/${version}/ | head -n 1 | grep -q "HTTP/1.[01] [23]..")
+        }
+        
+        elapsedTime=0
+        checkIfPresent
+        
+        # Allow 20 minutes for the file to appear in artifactory
+        while [ $? -ne 0 ] && [ $elapsedTime -lt 1200 ]; do
+            sleep 10;
+            let "elapsedTime+=10"
+            checkIfPresent
+        done
+        
+        exit $?
+    >>>
+    
+    output {
+        String publishedVersion = version
+    }
+}
+
+task create_update_dependency_command {
+    String dependencyName
+    String newVersion
+    String dependencyFilePath = "build.sbt"
+    
+    command {
+        echo "sed -i '' \"s/${dependencyName}[[:space:]]=.*/${dependencyName} = \\\"${newVersion}\\\"/g\" ${dependencyFilePath}"
+    }
+    
+    output {
+      String updateCommand = read_string(stdout())
+    }
+}
+
+task intVersionPrep {
+    String organization
+    String repo
+    String file
+    String regexPrefix
+    String updateCommandTemplate
+    
+    String bash_rematch = "{BASH_REMATCH[1]}"
+    
+    command {
+        curl -o versionFile https://raw.githubusercontent.com/${organization}/${repo}/develop/${file}
+        regex="${regexPrefix}\"([0-9]+)\""
+        
+        if [[ $(cat versionFile) =~ $regex ]]
+        then
+            version="$${bash_rematch}"
+            echo $version > version
+        else
+            exit 1
+        fi
+    }
+    
+    output {
+        Int version = read_int("version")
+        Int nextVersion = version + 1
+        String updateCommand = sub(updateCommandTemplate, "<<VERSION>>", nextVersion)
+    }
+}
+
+task decimalVersionPrep {
+    String organization
+    String repo
+    String file
+    String regexPrefix
+    String updateCommandTemplate
+    
+    String bash_rematch = "{BASH_REMATCH[1]}"
+    command {
+        curl -o versionFile https://raw.githubusercontent.com/${organization}/${repo}/develop/${file}
+        regex="${regexPrefix}\"([0-9]+\.[0-9]+)\""
+        
+        if [[ $(cat versionFile) =~ $regex ]]
+        then
+            version="$${bash_rematch}"
+            echo $version > version
+        else
+            exit 1
+        fi
+    }
+    
+    output {
+        Float version = read_float("version")
+        Float nextVersion = version + 0.01
```

Comment:
That'll work until we hit version `.100` 😄 

It looks like this and `intVersionPrep` could be unified:

```
computer:~ user$ echo 3 | perl -ne 'if (/^([0-9]+\.)?([0-9]+)$/) { $incr = $2 + 1; print "$1$incr\n" }'
4
computer:~ user$ echo 3.1 | perl -ne 'if (/^([0-9]+\.)?([0-9]+)$/) { $incr = $2 + 1; print "$1$incr\n" }'
3.2
computer:~ user$ echo 3.14 | perl -ne 'if (/^([0-9]+\.)?([0-9]+)$/) { $incr = $2 + 1; print "$1$incr\n" }'
3.15
computer:~ user$ echo 0.1 | perl -ne 'if (/^([0-9]+\.)?([0-9]+)$/) { $incr = $2 + 1; print "$1$incr\n" }'
0.2
computer:~ user$ echo 0.11 | perl -ne 'if (/^([0-9]+\.)?([0-9]+)$/) { $incr = $2 + 1; print "$1$incr\n" }'
0.12
```